cobalt/browser/metrics/cobalt_metrics_log_uploader.cc

```diff
@@ -20,12 +20,25 @@
 #include "components/metrics/log_decoder.h"
 #include "components/metrics/metrics_log_uploader.h"
 #include "third_party/metrics_proto/chrome_user_metrics_extension.pb.h"
+#include "third_party/metrics_proto/cobalt_uma_event.pb.h"
 #include "third_party/metrics_proto/reporting_info.pb.h"
 
 namespace cobalt {
 namespace browser {
 namespace metrics {
 
+// Helper method to create a trimmed down and sanitized version of the UMA
+// proto for Cobalt
+void PopulateCobaltUmaEvent(
+    const ::metrics::ChromeUserMetricsExtension& uma_proto,
+    const ::metrics::ReportingInfo reporting_info_proto,
+    CobaltUMAEvent& cobalt_proto) {
```

Comment:
Any reason to use an output parameter instead of a return value for this function?

Reply:
Not particularly. I think I reached for it this time as the "DecodeLogData" call above it uses the same pattern.